CRUD/Peca.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CRUD
+{
+    public class Peca
+    {
+        public Peca(string Nome, string Categoria, string Descricao, int Estoque) 
+        {
+            this.Nome= Nome;
```

Comment:
Corrige os nomes dos parâmetros para 'camelCase' e retira o this. das propriedades que recebem o valor de entrada.